src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,3 +85,118 @@ def load_arguments(self, _):
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# Cassandra
+    with self.argument_context('cosmosdb cassandra keyspace') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('keyspace_name', options_list=['--keyspace', '-n'], help="Keyspace name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+    with self.argument_context('cosmosdb cassandra table') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('keyspace_name', options_list=['--keyspace', '-k'], help="Keyspace name")
+        c.argument('table_name', options_list=['--table', '-n'], help="Table name")
+        c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL (https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live)')
+        c.argument('schema', type=shell_safe_json_parse, completer=FilesCompleter(), help='Schema, you can enter it as a string or as a file, e.g., --schema @schema-file.json or ' + CASSANDRA_SCHEMA_EXAMPLE)
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# Gremlin
+    with self.argument_context('cosmosdb gremlin database') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-n'], help="Database name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+    with self.argument_context('cosmosdb gremlin graph') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-d'], help="Database name")
+        c.argument('graph_name', options_list=['--graph', '-n'], help="Graph name")
+        c.argument('partition_key_path', options_list=['--part'], help='Partition Key Path, e.g., \'/properties/name\'')
+        c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL (https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live)')
+        c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_GREMLIN_INDEXING_POLICY_EXAMPLE)
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# Table
+    with self.argument_context('cosmosdb table') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('table_name', options_list=['--table', '-n'], help="Table name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
```

Comment:
For help message, Offer throughput of table under a Azure Cosmos DB database account.

Please apply to other throughput argument, too.